### PR TITLE
Decouple format from outfile

### PR DESCRIPTION
**Pull request by michaelhoefer**
Decouple format from outfile.  Should be able to specify both.

### CODE REVIEW

File: src/commands/scanner/run.ts [hunk 1]
```diff
@@ -79,14 +79,11 @@ export default class Run extends ScannerCommand {
     format: flags.enum({
       char: 'f',
       description: messages.getMessage('flags.formatDescription'),
-      options: [OUTPUT_FORMAT.XML, OUTPUT_FORMAT.JUNIT, OUTPUT_FORMAT.CSV, OUTPUT_FORMAT.TABLE],
-      default: OUTPUT_FORMAT.TABLE,
-      exclusive: ['outfile']
```
Comment (jfeingold35) [inline src/commands/scanner/run.ts:84]: If you're getting rid of the exclusivity, surely you still need some kind of validation, right?

Comment (michaelhoefer) [reply src/commands/scanner/run.ts:84]: What additional validation do you think is needed that the exclusivity formerly provided?

Comment (jfeingold35) [reply src/commands/scanner/run.ts:84]: I imagine you'd want to disallow `--format csv --outfile something.xml` or `--format xml --outfile something.csv`, right? There's probably a few other combinations that are invalid.

Comment (michaelhoefer) [reply src/commands/scanner/run.ts:84]: I think I would lean toward liberty on that front.  If someone really wants to write xml to a .csv file, they may have a weird but good reason for that.  It doesn't introduce any risk that I can see.  Your OS lets you rename any file type to any extension.

Comment (jfeingold35) [reply src/commands/scanner/run.ts:84]: Fair enough, but perhaps it's worth logging a warning?

Comment (michaelhoefer) [reply src/commands/scanner/run.ts:84]: Cool.  Will add that.

**Pull request merged**